src/main/groovy/nextflow/container/ShifterBuilder.groovy

```diff
@@ -35,7 +35,7 @@ class ShifterBuilder extends ContainerBuilder {
         function shifter_img() {
           local cmd=$1
           local image=$2
-          shifterimg -v $cmd $image |  awk -F: '$0~/status/{gsub("[\\", ]","",$2);print $2}\'
+          shifterimg -v $cmd $image |  awk -F: '$0~/status":/{gsub("[\\", ]","",$2);print $2}\'
```

Comment:
I'm not sure about the first `"`. Why is it needed ?

Reply:
It's part of the Shifter string:

```
"status": "READY"
```

without it, there will be no match at all.

Reply:
I see, good. If so we may use `"status":` .. to make it a bit more readable. What do you think ?

Reply:
Yes, I think that would be nice.